port/mmap.h

```diff
@@ -67,4 +68,20 @@ class MemMapping {
   static MemMapping AllocateAnonymous(size_t length, bool huge);
 };
 
+template <typename T>
```

Comment:
It would be great to have a simple documentation for this class, especially about what extra benefits it has over the base class. Thanks!

Reply:
I'll add a short comment